tests/main/services-refresh-mode/task.yaml

```diff
@@ -3,7 +3,7 @@ summary: "Check that refresh-modes works"
 # takes >1.5min to run
 backends: [-autopkgtest]
 
-kill-timeout: 12m
+kill-timeout: 10m
```

Comment:
(nitpick) here the time decreases by 2min, is that intentional?

Reply:
Yes, see the bottom of PR desc: The test services-refresh-mode initially was in 5 minutes, then moved to 12 but when tested with 10m it works fine.